src/Control/Distributed/Process/Internal/Types.hs

```diff
@@ -137,7 +137,7 @@ import GHC.Generics
 newtype NodeId = NodeId { nodeAddress :: NT.EndPointAddress }
   deriving (Eq, Ord, Typeable, Data, Generic)
 instance Binary NodeId where
-instance NFData NodeId
+instance NFData NodeId where rnf (NodeId a) = rnf a `seq` ()
```

Comment:
We should depend on deepseq>=1.4 and drop the `where rnf (NodeId a) = rnf a seq ()`.

http://hackage.haskell.org/package/deepseq-1.4.1.1/docs/Control-DeepSeq.html#t:NFData

Unless I'm misunderstanding the point of this change?